docs/pages/access-controls/guides/hardware-key-support.mdx

```diff
@@ -76,14 +76,14 @@ like `tctl edit`. With touch required, hardware key support provides better secu
 </Admonition>
 
 - Install a smart card driver for you operating system. Teleport clients will connect to your YubiKey through the smart card driver to generate keys and perform cryptographic operations.
-  - MacOS and Windows both ship with smart card drivers.
+  - macOS and Windows both ship with smart card drivers.
```

Comment:
Maybe we could add `MacOS` to `flagWords` in `docs/cspell.json`? I'm not entirely familiar with how it works, but based on the docs it should do the job.